src/genesis.rs

```diff
@@ -179,6 +210,83 @@ impl Genesis {
         Ok(())
     }
 
+    pub fn create_client_accounts(
+        &mut self,
+        number_of_clients: usize,
+        bench_tps_args: &[String],
+        target_lamports_per_signature: u64,
+        config_dir: &Path,
+        deploy_method: &DeployMethod,
+        solana_root_path: &Path,
+    ) -> Result<(), Box<dyn Error>> {
+        if number_of_clients == 0 {
+            return Ok(());
+        }
+        let client_accounts_file = config_dir.join("client-accounts.yml");
+
+        info!("generating {number_of_clients} client account(s)...");
+
+        let children: Result<Vec<_>, _> = (0..number_of_clients)
+            .map(|i| {
+                info!("client account: {i}");
+                let mut args = Vec::new();
+                let account_path = config_dir.join(format!("bench-tps-{i}.yml"));
+                args.push("--write-client-keys".to_string());
+                args.push(account_path.into_os_string().into_string().map_err(|err| {
+                    std::io::Error::new(
+                        std::io::ErrorKind::InvalidData,
+                        format!("Invalid Unicode data in path: {:?}", err),
+                    )
+                })?);
+                args.push("--target-lamports-per-signature".to_string());
+                args.push(target_lamports_per_signature.to_string());
+
+                if !bench_tps_args.is_empty() {
+                    args.extend(bench_tps_args.to_owned());
+                }
+
+                let executable_path = if let DeployMethod::ReleaseChannel(_) = deploy_method {
+                    solana_root_path.join("solana-release/bin/solana-bench-tps")
+                } else {
+                    solana_root_path.join("farf/bin/solana-bench-tps")
+                };
+
+                Self::create_client_account(&args, &executable_path)
+            })
+            .collect();
+
+        for child in children? {
+            let output = child.wait_with_output()?;
+            if !output.status.success() {
+                return Err(output.status.to_string().into());
+            }
+        }
+
+        let progress_bar = new_spinner_progress_bar();
+        progress_bar.set_message(format!("{WRITING}Writing client accounts..."));
```

Comment:
it is for combining all `bench-tps-{i}.yml` accounts into `client-accounts.yml` for genesis. it can take a while if there are a lot of client accounts. could def put in for whole function.

Reply:
Ah gotcha, thanks!